packages/provider-cosmoskit/README.md

```diff
@@ -0,0 +1,114 @@
+# abstractjs
+This repository holds the type declarations and interfaces for our smart contracts in CosmWasm.
+Autogenerated contracts lie in [generated](generated) and those that are "fixed" are placed in [src/contracts](src/contracts).
+
+
+## Scripts
+- `pnpm build`: Generate the types
+- `pnpm fix`: Temporary fixes for the autogenerated contracts (TODO)
+- `pnpm format`: Run prettier
+- `pnpm lint`: Run linter
+- `pnpm lint:fix`: Attempt linter fixes
+
+## Getting Started
+1. Install core
+```bash
+pnpm install @abstract-money/core
+```
+2. Initialize the `AbstractQueryClient` for the chain on which you're using Abstract.
+```ts
+import { AbstractQueryClient } from '@abstract-money/core'
+
+const CHAIN_NAME = "juno"
+const getAbstract = async () => {
+  const abstractQueryClient = await AbtsractQueryClient.connectToChain(CHAIN_NAME)
+}
+```
+
+
+## Abstract API
+1. Install dependencies
+```bash
+pnpm install graphql-request@5
+```
+
+### Typescript
+To allow for strongly-typing the schemas, we recommend setting up code generation.
+
+See https://www.apollographql.com/docs/ios/code-generation/codegen-configuration/ for alternative options.
+
+The following uses the-guild's graphql codegen as a typescript preset. See https://the-guild.dev/graphql/codegen/plugins/presets/preset-client for client options.
+
+1. Install dependencies
+```bash
+pnpm install -D @graphql-codegen/cli @graphql-codegen/client-preset @graphql-typed-document-node/core
+```
+2. Setup `codegen.yml`. This allows for auto type generation.
+See
+```yml
+schema: "<API_URL>" # This can be the API URL or the schema document
+
+documents: ['./src/**/*.{tsx,ts}', '!src/__generated__/gql/**/*']
+generates:
+  ./src/__generated__/gql/:
+    preset: client
+    presetConfig:
+      gqlTagName: gql
+      immutableTypes: true
+    config:
+      immutableTypes: true
+      useTypeImports: true
+      scalars:
+        # Temporarily map to any, see DAT-16
+        JSON: any
+        Semver: string
+        Bech32Address: string
```

Comment:
```suggestion
        Bech32Address: string

ignoreNoDocuments: true
```

Reply:
oh those are completely wrong as they came from the `core` copy, will remove them.